UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -728,6 +728,27 @@ class BTPayPalClient_Tests: XCTestCase {
     }
     
 
+    func testIsiOSAppSwitchAvailable_whenApplicationCanOpenPayPalInAppURL_returnsTrue() {
```

Comment:
Instead of asserting that the function `isPayPalAppInstalled` returns some boolean, we rather want our unit tests to assert on our public API. Implementation details (such as if the app installed check lives in a method or not) should be able to change without breaking our unit tests.

So this test should assert on the end result behavior, which is that the API POST request to `/v1/paypal_hermes/create_payment_resource` _does_ include the app switch params if the app is installed, and does _not_ include the app switch params if the app is not installed.

Reply:
Thanks for the feedback! I've made changes to test the endpoints